engine/src/main/resources/rest/messages.properties

```diff
@@ -216,8 +216,11 @@ user.passwords.same=New password has to be different from old password
 user.status.invalid=User status should be ''active'' or ''inactive''
 
 #labels
-labelRequest.titles.required=No languages specified
+labelRequest.key.required=the key is required
+labelRequest.key.invalidSize=the key is longer than 50 characters.
+labelRequest.key.invalidCharacters=the key contains invalid characters. Only alphanumeric characters and the underscore ''_'' are allowed.
```

Comment:
Remove "the" before the word underscore.

Reply:
Done